diff --git a/source/LibRender2/LibRender2.csproj b/source/LibRender2/LibRender2.csproj
index d361fe28e..07ecf1b76 100644
--- a/source/LibRender2/LibRender2.csproj
+++ b/source/LibRender2/LibRender2.csproj
@@ -20,6 +20,7 @@
     <EmbeddedResource Include="..\..\assets\Shaders\shadow_depth.vert" LogicalName="LibRender2.shadow_depth.vert" />
     <EmbeddedResource Include="..\..\assets\Shaders\shadow_depth.frag" LogicalName="LibRender2.shadow_depth.frag" />
     <EmbeddedResource Include="..\..\assets\Shaders\cluster_culling.comp" LogicalName="LibRender2.cluster_culling.comp" />
+    <EmbeddedResource Include="..\..\assets\Shaders\RealSky.comp" LogicalName="LibRender2.RealSky.comp" />
   </ItemGroup>
 
   <ItemGroup>
